<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkerDialog</class>
 <widget class="QDialog" name="MarkerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>时标</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resourcefile.qrc">
    <normaloff>:/Icon/Icon/wave-6.png</normaloff>:/Icon/Icon/wave-6.png</iconset>
  </property>
  <widget class="QFrame" name="frame_8">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>261</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="MarkerPeriod_Label">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="MarkerPeriod_Dsb">
          <property name="minimumSize">
           <size>
            <width>82</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>82</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="MarkerPeriodUnit_Cmb">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Outline</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="MarkerPeriod_Dial"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="StandBy_pushBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>82</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 1px solid #000000;
    border-radius: 5px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #dedede, stop: 0.5 #434343,
                                   stop: 0.51 #000000, stop: 1 #656a6d);
    color: #FFFFFF;
    font: bold 14px;
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #cfcccc, stop: 0.5 #333232,
                                   stop: 0.51 #000000, stop: 1 #585c5f);
    color: #00CC00;
}
QPushButton:flat {
    border: none;
}</string>
   </property>
   <property name="text">
    <string>STANDBY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OutputOn_pushBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>89</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 1px solid #000000;
    border-radius: 5px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #dedede, stop: 0.5 #434343,
                                   stop: 0.51 #000000, stop: 1 #656a6d);
    color: #FFFFFF;
    font: bold 14px;
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #cfcccc, stop: 0.5 #333232,
                                   stop: 0.51 #000000, stop: 1 #585c5f);
    color: #00CC00;
}
 
QPushButton:flat {
    border: none;
}</string>
   </property>
   <property name="text">
    <string>OUTPUTON</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <widget class="QLabel" name="MarkerAmp_Label">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="MarkerAmp_Cmb">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QLabel" name="MarkerWaveNo_Label">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="MarkerWaveNo_Cmb">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resourcefile.qrc"/>
 </resources>
 <connections/>
</ui>
